build/commands/lib/config.js

```diff
@@ -177,6 +176,35 @@ Config.prototype.isOfficialBuild = function () {
   return this.buildConfig === 'Release'
 }
 
+Config.prototype.isStableChannel = function () {
+  return this.channel === '' || this.channel === 'release'
+}
+
+Config.prototype.isBetaChannel = function () {
+  return this.channel === 'beta'
+}
+
+Config.prototype.isNightlyChannel = function () {
+  return this.channel === 'nightly'
+}
+
+Config.prototype.isDevChannel = function () {
+  return this.channel === 'dev' || this.channel === 'development'
+}
+
+Config.prototype.isBraveReleaseBuild = function () {
+  const npm_brave_relese_build = getNPMConfig(['is_brave_release_build'])
+  if (npm_brave_relese_build !== undefined) {
+    assert(npm_brave_relese_build === '0' || npm_brave_relese_build === '1',
+      'Bad is_brave_release_build npm value (should be 0 or 1)')
+    return npm_brave_relese_build === '1'
+  }
+
+  // TODO: Replace it to `this.isOfficialBuild() && !this.isDevChannel()`
```

Comment:
see https://github.com/brave/brave-core/pull/11549/files#r776507382
what I think we want here is return `this.isOfficialBuild() && this.channel !== 'development'`

Reply:
but it shouldn't be a TODO, this is what we want right now

Reply:
In fact channel [can't be](https://github.com/brave/brave-core/blob/master/build/config.gni#L68) `development` for is_official_build.
Returned `this.isOfficialBuild()`.
As an alternative we could move PR's builder to `dev` channel and check `channel != 'dev'` here.